pages/programs/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { ProgramsList } from '../../lib/components/programs-list';
+import { Text, Container, Flex } from '@chakra-ui/react';
+import OSLHeader from '../../lib/components/header';
+
+const Programs: React.FC = () => {
+  return (
+    <Container maxW={'container.lg'} mb="100px" fontSize="24px" paddingTop="24px">
+      <Flex flexDirection="column" align="flex-start">
```

Comment:
❓ Could we use a Chakra Stack here?

Reply:
I learned that the stack isn't great for responsiveness and I am pushing to use Flex everywhere because it is just the best!